lte/gateway/c/sctpd/src/BUILD.bazel

```diff
@@ -24,6 +24,9 @@ cc_binary(
         ":config",
         ":sctpd_downlink_impl",
         ":sctpd_event_handler",
+        "//lte/protos:mconfigs_cpp_proto",
+        "//orc8r/gateway/c/common/config:mconfig_loader",
+        "//orc8r/gateway/c/common/sentry:sentry_wrapper",
```

Comment:
This is so much more intuitive than the cmake file. :smile:

Reply:
Yeah.. CMake exposes all sorts of the guts of building and linking as part of the configuration files - when that is absolutely not necessary and a better abstraction talks about e.g. Dependencies for the most part (which is the approach of Bazel / Buck).  This is precisely one of our arguments on Bazel - it is *hard* to contribute to our C/C++ today given the CMake situation (which admittedly is worse than it needs to be, if we were to completely fix all the CMake non-best-practices).  Bazel offers a minimum API and a "blessed path" primarily - so it's easy to replicate and do things the right way and you need not get into the guts of compilation.  /rant